### DISCUSSION TITLE
Firefly-III Importer - Merge columns

### DISCUSSION THREAD

**Discussion by JeanMerand** [firefly-iii/firefly-iii#6668] [Firefly III] [answered]
Hi,

I did install Firefly-III on my unRAID server, and it works well.

To get data, I import via CSV file downloaded on website of my bank (I also tried Nordigen but I miss a lot of things to compare to CSV file).

I'm wondering if there is a way to merge columns when importing. 
For example I would like to have info of Column 2 and Column 5 merged together with the separator (space) and go into the filed "Notes".

Is it possible to do it by changing or editing a setup file somewhere in Firefly-III Importer ?
Or maybe I have to manage it via a rule ?

Thank you.

**Comment by JC5** [comment, marked as answer]
You can set both columns to "notes", they will be merged automatically.

https://docs.firefly-iii.org/data-importer/usage/roles/#notes

**Reply by JeanMerand** [reply]
Ah, that's perfect, sorry I didn't see it in the documentation.
Thank you.
